frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/eviction-strategy-modal/eviction-strategy-modal.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  createModalLauncher,
+  ModalTitle,
+  ModalBody,
+  ModalComponentProps,
+} from '@console/internal/components/factory';
+import { ModalFooter, ModalFooterProps } from '../../modal/modal-footer';
```

Comment:
please try to keep the import order convention (vendor first, then @console/xxx, then relative paths).